lua/packer.lua

```diff
@@ -80,13 +85,36 @@ local config_defaults = {
 local config = vim.tbl_extend('force', {}, config_defaults)
 local plugins = nil
 local rocks = nil
+local plugins_snapshot = {}
+
+--- Loads and parse snapshot file in the global variable plugins_snapshot
+--@param filename path/to/snapshot
+packer.load_snapshot = function (filename)
+  local f_snapshot, err = io.open(filename, 'r+')
+
+  if err ~= nil then
+    print(err)
+    log.info(err)
+  else
+    for line in f_snapshot:lines() do
+      local short_name, commit = unpack(vim.split(line, ' '))
```

Comment:
Do I correctly understand that the snapshots save plugin names and commits, but not e.g. configuration other than the commit?

Reply:
> Do I correctly understand that the snapshots save plugin names and commits, but not e.g. configuration other than the commit?

Yes, since configs aren't really part of the plugin state I thought it was better to leave it out, or do you think it might be better to back all up in the snapshot (i.e. `config`, `requires` ecc...)

Reply:
For a first pass on the feature, I think just snapshotting commits is fine - we just need to be clear in the docs that we don't save other config information.